lib/DataStore/Utility/Mongo.php

```diff
@@ -1,127 +0,0 @@
-<?php
-
-namespace Aol\Transformers\DataStore\Utility;
-
-use Aol\Transformers\DataStore\UtilityInterface;
-use Aol\Transformers\Utility;
-use MongoDate;
-use MongoId;
-
-class Mongo extends Utility implements UtilityInterface
-{
-	/**
-	 * @param \MongoId|null $id
-	 * @return string
-	 */
-	public function idToApp($id)
-	{
-		return (string)$id;
-	}
-
-	/**
-	 * @param string $id
-	 * @return MongoId|null
-	 */
-	public function idToExt($id)
-	{
-		return empty($id) ? null : new MongoId($id);
-	}
-
-	/**
-	 * @param \MongoDate|null $date
-	 * @return \DateTime|null
-	 */
-	public function dateTimeToApp($date)
-	{
-		return $date instanceof MongoDate ? new \DateTime('@' . $date->sec) : null;
-	}
-
-	/**
-	 * @param \DateTime $date
-	 * @return MongoDate|null
-	 */
-	public function dateTimeToExt($date)
-	{
-		if (empty($date)) {
-			return null;
-		}
-
-		$date = $date instanceof \DateTime ? $date->getTimestamp() : strtotime($date);
-
-		return new MongoDate($date);
-	}
-
-	/**
-	 * @param mixed $date
-	 * @return \DateTime|null
-	 */
-	public function dateToApp($date)
-	{
-		return $this->dateTimeToApp($date);
-	}
-
-	/**
-	 * @param \DateTime|null $date
-	 * @return MongoDate|null
-	 */
-	public function dateToExt($date)
-	{
-		return $this->dateTimeToExt($date);
-	}
-
-	// Additional methods
-	///////////////////////////////////////////////////////////////////////////
-
-	/**
-	 * Get created DateTime from a MongoId object.
-	 *
-	 * @param MongoId $id
-	 * @return \DateTime
-	 */
-	public function getDateFromMongoId(MongoId $id)
-	{
-		$date_time = new \DateTime('@' . $id->getTimestamp());
-		$date_time->setTimezone(new \DateTimeZone(date_default_timezone_get()));
-
-		return $date_time;
-	}
-
```

Comment:
Perhaps for now leaving in `MongoTrait` makes sense, since they're universally useful. Looking at a current use case, I'm using `getDateFromMongoId` in a transformer's `afterApp`, for instance. 

I also have two more general utility methods that could be included, because they kinda fit this pattern of transform/un-transform as well, called `esacpeMongoKeys`/`unescapeMongoKeys`, that are also used in those transformer hooks.